app/src/lib/get-os.ts

```diff
@@ -16,3 +16,24 @@ export function getOS() {
     return `${OS.type()} ${OS.release()}`
   }
 }
+
+/** See the OS we're currently running on is at least Mojave. */
+export function isMojaveOrLater() {
+  if (__DARWIN__) {
+    const parser = new UAParser()
+    const os = parser.getOS()
+
+    if (os.version == null) {
+      return false
+    }
+
+    const parts = os.version.split('.')
+    if (parts.length < 2) {
```

Comment:
I don't think `parts.length` should ever be anything other than `3` so we can change this to `parts !== 3`